trace/trace.go

```diff
@@ -353,6 +356,22 @@ func (sc SpanContext) IsValid() bool {
 	return sc.HasTraceID() && sc.HasSpanID()
 }
 
+// IsRemote indicates whether the SpanContext represents a remotely-created Span.
+func (sc SpanContext) IsRemote() bool {
+	return sc.remote
+}
+
+// WithRemote returns a new SpanContext with the remote property replaced.
```

Comment:
Making it explicit it is a copy of sc will be helpful.

```suggestion
// WithRemote returns a copy of sc with the Remote property set to the remote.
```